src/Rules/Deprecations/SymfonyCmfRouteObjectInterfaceConstantsRule.php

```diff
@@ -26,6 +26,9 @@ public function getNodeType(): string
 
     public function processNode(Node $node, Scope $scope): array
     {
+        if (!class_exists(SymfonyRouteObjectInterface::class)) {
+            return [];
+        }
```

Comment:
This is causing `SymfonyCmfRouteObjectInterfaceConstantsRuleTest` to fail on ^9.0 and I'm not sure why.